base/libuv.jl

```diff
@@ -133,7 +133,8 @@ function uv_return_spawn end
 function uv_asynccb end
 function uv_timercb end
 
-function reinit_stdio()
+reinit_stdio() = _reinit_stdio()
+function _reinit_stdio()
```

Comment:
What in the world does this do?

Reply:
Well, this PR is about "more small changes for trimming" -- and this adds a change we can later trim? 🤔  😂

Reply:
Ah, I think we did this at one point so we could disable Base's call to it, but still be able to easily call it ourselves later. But it looks like we don't need this anymore.